ACPs/77-reinventing-subnets/README.md

```diff
@@ -505,66 +513,65 @@ def cost_over_time(V:int, T:int, x:int, Δt: int) -> int:
+The expectation is that most users will not interact with node clients or track when or how much they need to top-up their validator account. Wallet providers will abstract away most of this process. For users who desire more convenience, L1-as-a-Service providers will abstract away all of this process.
 
 ## Backwards Compatibility
 
-This new design for Subnets proposes a large rework to all Subnet-related mechanics. Rollout should be done on a going-forward basis to not cause any service disruption for live Subnets. All current Subnet Validators will be able to continue validating both the Primary Network and whatever Subnets they are validating.
+This new design for Subnets proposes a large rework to all L1-related mechanics. Rollout should be done on a going-forward basis to not cause any service disruption for live Subnets. All current Subnet validators will be able to continue validating both the Primary Network and whatever Subnets they are validating.
 
 Any state execution changes must be coordinated through a mandatory upgrade. Implementors must take care to continue to verify the existing ruleset until the upgrade is activated. After activation, nodes should verify the new ruleset. Implementors must take care to only verify the presence of 2000 $AVAX prior to activation.
 
 ### Deactivated Transactions
 
 - P-Chain
+
   - `TransformSubnetTx`
 
     After this ACP is activated, Elastic Subnets will be disabled. `TransformSubnetTx` will not be accepted post-activation. As there are no Mainnet Elastic Subnets, there should be no production impact with this deactivation.
 
 ### New Transactions
 
 - P-Chain
-  - `ConvertSubnetTx`
-  - `RegisterSubnetValidatorTx`
-  - `SetSubnetValidatorWeightTx`
-  - `DisableSubnetValidatorTx`
-  - `IncreaseSubnetValidatorBalanceTx`
+  - `ConvertSubnetToL1Tx`
+  - `RegisterL1ValidatorTx`
+  - `SetL1ValidatorWeightTx`
+  - `DisableL1ValidatorTx`
+  - `IncreaseL1ValidatorBalanceTx`
 
 ## Reference Implementation
 
 A full reference implementation has not been provided yet. It will be provided once this ACP is considered `Implementable`.
 
 ## Security Considerations
 
-This ACP significantly reduces the cost of becoming a Subnet Validator. This can lead to a large increase in the number of Subnet Validators going forward. Each additional Subnet Validator adds consistent RAM usage to the P-Chain. However, this should be appropriately metered by the continuous fee mechanism outlined above.
+This ACP significantly reduces the cost of becoming an L1 validator. This can lead to a large increase in the number of L1 validators going forward. Each additional validator adds consistent RAM usage to the P-Chain. However, this should be appropriately metered by the continuous fee mechanism outlined above.
```

Comment:
```suggestion
This ACP introduces Avalanche Layer 1s, a new network type that costs significantly less from Avalanche Subnets. This can lead to a large increase in the number of networks and, by extension, the number of validators. Each additional validator adds consistent RAM usage to the P-Chain. However, this should be appropriately metered by the continuous fee mechanism outlined above.
```